doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -0,0 +1,104 @@
+.. _building_nrf54l:
+
+Building and programming with nRF54L15 DK
+#########################################
+
+This guide provides instructions on how to build and program the nRF54L15 development kit.
+Whether you are working with single or multi-image builds, the following sections will guide you through the necessary steps.
+
+Depending on the sample, you must program only the application core or both the FLPR and the application core.
+Additionally, the process will differ based on whether you are working with a single-image or multi-image build.
+
+Building for the application core
+*********************************
+
+For instructions on building for the application core only, see how to :ref:`program an application <programming_cmd>`.
+
+Building for the application and FLPR core
+******************************************
+
+This section outlines how to build and program for both the application and FLPR cores, covering separate builds and sysbuild configurations.
+
+Separate images
+---------------
+
+To build and program the application sample and the VPR sample as separate images, follow these steps:
```

Comment:
So far the VPR is referred to as FLPR in this document. Introducing a new term might be confusing for a reader.
```suggestion
To build and program the application sample and the FLPR sample as separate images, follow these steps:
```